src/index.js

```diff
@@ -0,0 +1,7 @@
+import app from './app';
+
+require('dotenv').config();
```

Comment:
If you want to use an import statement here, you can do `import {} from 'dotenv/config'` and it'll achieve the same thing without having use a require statement

Reply:
Nice! Will fix.